src/utils/functions.js

```diff
@@ -1,9 +1,46 @@
-import { cache } from './util'
+import { cache, newPreCommandServiceOptions } from './util'
+import commands from 'iroha-helpers/lib/commands'
 
 function isLoggedIn () {
   return !!cache.username
 }
 
+function preDefinedPrivateKey () {
```

Comment:
It's function, but return data. It's okay to use such name for such functions.

Reply:
But i think better to use predefined (as one word) not preDefined (as two words)